### ISSUE TITLE
 merge substrate classes from PLD and MOVPE and place it under IKZ category

### ISSUE THREAD

**Issue by aalbino2** [FAIRmat-NFDI/AreaA-data_modeling_and_schemas#125]
I will attemp a merge of the classes and then @hampusnasstrom will carefully revise if everything is included

**Comment by aalbino2** [comment]
to close this issue, can you remove the PVD folder in the root of AreaA repo @hampusnasstrom ? Thanks

**Comment by hampusnasstrom** [comment]
I still have general things in there. Did we decide on creating an "old" folder or something where I can move it?

**Comment by aalbino2** [comment]
no, we didn't... you could move it elsewhere like in your personal one?

If you htink this is okay, go for that. These old stuff would only pollute the repo - in view of publishing it in a JOSS

There is also base_sections/nomad-material.processing to be deleted for example

**Comment by aalbino2** [comment]
@budschi let's meet with @ThomasTSC and Jeremy to discuss the Substrate class where you guys agree on what to record
